src/EnergyPlus/HVACMultiSpeedHeatPump.cc

```diff
@@ -1944,80 +1874,54 @@ namespace HVACMultiSpeedHeatPump {
-        using PlantUtilities::ScanPlantLoopsForObject;
-        using PlantUtilities::SetComponentFlowRate;
-        using Psychrometrics::PsyRhoAirFnPbTdbW;
-        using ScheduleManager::GetCurrentScheduleValue;
-        using SteamCoils::SimulateSteamCoilComponents;
-        using WaterCoils::GetCoilMaxWaterFlowRate;
-        using WaterCoils::SimulateWaterCoilComponents;
-
         // SUBROUTINE LOCAL VARIABLE DECLARATIONS:
         static constexpr std::string_view RoutineName("InitMSHeatPump");
-        int InNode;     // Inlet node number in MSHP loop
-        int OutNode;    // Outlet node number in MSHP loop
-        int ZoneInNode; // Zone inlet node number in the controlled zone for MSHP
-        Real64 RhoAir;  // Air density at InNode
+        Real64 RhoAir; // Air density at InNode
 
         Real64 QSensUnitOut; // Output of MSHP system with coils off
         Real64 PartLoadFrac; // Part-load ratio
         int ZoneNum;
-        int i;                 // Index to speed
-        int NumOfSpeedCooling; // Number of speeds for cooling
-        int NumOfSpeedHeating; // Number of speeds for heating
-        Real64 DeltaMassRate;  // Difference of mass flow rate between inlet node and system outlet node
+        int i;                // Index to speed
+        Real64 DeltaMassRate; // Difference of mass flow rate between inlet node and system outlet node
 
-        int ZoneInSysIndex(0);                            // number of zone inlet nodes counter in an airloop
         int NumAirLoopZones(0);                           // number of zone inlet nodes in an air loop
-        int ZoneInletNodeNum(0);                          // zone inlet nodes node number
         Real64 SumOfMassFlowRateMax(0.0);                 // the sum of mass flow rates at inlet to zones in an airloop
         Real64 CntrlZoneTerminalUnitMassFlowRateMax(0.0); // Maximum mass flow rate through controlled zone terminal unit
-        bool errFlag;
-        Real64 rho;    // local fluid density
-        Real64 MdotHR; // local temporary for heat recovery fluid mass flow rate (kg/s)
+        Real64 rho;                                       // local fluid density
+        Real64 MdotHR;                                    // local temporary for heat recovery fluid mass flow rate (kg/s)
         Real64 ZoneLoadToCoolSPSequenced;
         Real64 ZoneLoadToHeatSPSequenced;
 
         bool ErrorsFound(false);        // flag returned from mining call
-        int SteamIndex(0);              // index of steam quality for steam heating coil
         Real64 mdot(0.0);               // local temporary for mass flow rate (kg/s)
         Real64 SteamDensity(0.0);       // density of steam at 100C, used for steam heating coils
         Real64 CoilMaxVolFlowRate(0.0); // coil fluid maximum volume flow rate
         Real64 QActual(0.0);            // coil actual capacity
-        int CoilAvailSchPtr(0);         // DX coil availability schedule pointer
```

Comment:
[src/EnergyPlus/HVACMultiSpeedHeatPump.cc:1996]:(style),[variableScope],The scope of the variable 'CoilAvailSchPtr' can be reduced.
[src/EnergyPlus/HVACMultiSpeedHeatPump.cc:1996]:(style),[unreadVariable],Variable 'CoilAvailSchPtr' is assigned a value that is never used.